package.json

```diff
@@ -39,8 +39,8 @@
     "cheerio": "0.20.0",
     "xmldom": "0.1.22",
     "logops": "1.0.0-alpha.7",
-    "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
-    "lwm2m-node-lib": "git://github.com/telefonicaid/lwm2m-node-lib.git#master",
+    "iotagent-node-lib": "git://github.com/m4n3dw0lf/iotagent-node-lib.git#master",
```

Comment:
If the PR is finally approved, this should revert to its original value just before merging.

Reply:
Yeah, but the lwm2m-node-lib need to be merged with the DTLS Implementation..